BE/GovernmentEntities/AsianJurisdiction/EasternAsiaGovernmentEntitiesAndJurisdictions.rdf

```diff
@@ -107,35 +100,30 @@
 	<owl:NamedIndividual rdf:about="&fibo-be-ge-easj;GovernmentOfThePeoplesRepublicOfChina">
 		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
 		<rdfs:label xml:lang="en">Government of the People&apos;s Republic of China</rdfs:label>
+		<rdfs:label xml:lang="zh">中华人民共和国</rdfs:label>
 		<skos:definition>unitary Marxist-Leninist single-party socialist republic in Eastern Asia, covering approximately 9.6 million square kilometers (3.7 million square miles), that is the world&apos;s third or fourth-largest country by area</skos:definition>
```

Comment:
I don't think we should get into areas of politics, and we have not for many other governments. Further it's confusing for constitution of the CCP with that of the country. That of the CCP in fact states (as of 2002 https://fas.org/irp/world/china/docs/const.html) "The Communist Party of China takes Marxism-Leninism, Mao Zedong 
Thought, Deng Xiaoping Theory and the important thought of Three 
Represents as its guide to action. "
The country's constitution http://www.npc.gov.cn/zgrdw/englishnpc/Constitution/2007-11/15/content_1372963.htm is more general "The People’s Republic of China is a socialist state under the people’s democratic dictatorship led by the working class and based on the alliance of workers and peasants."
Also the definition of the Government should not make semi-factual statements about irrelevant matters related to the region/country and its size.